guides/source/configuring.md

```diff
@@ -237,9 +237,10 @@ Every Rails application comes with a standard set of middleware which it uses in
 
     ```ruby
     Rails.application.config.hosts = [
-      IPAddr.new("0.0.0.0/0"), # All IPv4 addresses.
-      IPAddr.new("::/0"),      # All IPv6 addresses.
-      "localhost"              # The localhost reserved domain.
+      IPAddr.new("0.0.0.0/0"),        # All IPv4 addresses.
+      IPAddr.new("::/0"),             # All IPv6 addresses.
+      "localhost",                    # The localhost reserved domain.
+      ENV["RAILS_DEVELOPMENT_HOSTS"]  # Additional comma separated hosts for development.
```

Comment:
```suggestion
      ENV["RAILS_DEVELOPMENT_HOSTS"]  # Additional comma-separated hosts for development.
```

Reply:
Also, this line reads a bit weird to me because we won't add the comma-separated string as-is to the `hosts` config. But I guess changing it to `*ENV["RAILS_DEVELOPMENT_HOSTS"].split("\n")` would probably read worse? I'm not sure. Maybe someone has a clever third option.

Reply:
How about something more like this:

> It is included in the development environment by default, with `Rails.application.config.hosts` including:
> * All IPv4 addresses (`IPAddr.new("0.0.0.0/0")`
> * …
> * Additional comma-separated hosts from `ENV["RAILS_DEVELOPMENT_HOSTS"]`

Reply:
Thanks for catching the missing dash! I've added that in and will let others weigh in on any larger wording change.